code-samples/serving/hello-world/helloworld-python/README.md

```diff
@@ -113,89 +113,83 @@ cd knative-docs/code-samples/serving/hello-world/helloworld-python
-                    env:
-                      - name: TARGET
-                        value: "Python Sample v1"
-        ```
-
-       Ensure that the container image value in `service.yaml` matches the container
-       you built in the previous step. Apply the configuration using `kubectl`:
-
-       ```bash
-       kubectl apply --filename service.yaml
-       ```
-
-=== "kn"
-
-       With `kn` you can deploy the service with
+### "yaml"
+Create a new file, `service.yaml` and copy the following service
+definition into the file. Make sure to replace `{username}` with your
+Docker Hub username or with the URL provided by your container registry
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+name: helloworld-python
+namespace: default
+spec:
+template:
+    spec:
+    containers:
+        - image: docker.io/{username}/helloworld-python
+        env:
+            - name: TARGET
+            value: "Python Sample v1"
+```
 
-       ```bash
-       kn service create helloworld-python --image=docker.io/{username}/helloworld-python --env TARGET="Python Sample v1"
-       ```
+Ensure that the container image value in `service.yaml` matches the container
+you built in the previous step. Apply the configuration using `kubectl`:
 
-       This will wait until your service is deployed and ready, and ultimately it
-       will print the URL through which you can access the service.
+```bash
+kubectl apply --filename service.yaml
+```
 
+### "kn"
+With `kn` you can deploy the service with
 
+```bash
+kn service create helloworld-python --image=docker.io/{username}/helloworld-python --env TARGET="Python Sample v1"
+```
 
+This will wait until your service is deployed and ready, and ultimately it
+will print the URL through which you can access the service.
 
-   During the creation of your service, Knative performs the following steps:
+During the creation of your service, Knative performs the following steps:
 
-   - Creates a new immutable revision for this version of the app.
-   - Network programming to create a route, ingress, service, and load balance
-     for your app.
-   - Automatically scales your pods up and down, including scaling down to zero
-     active pods.
+- Creates a new immutable revision for this version of the app.
+- Network programming to create a route, ingress, service, and load balance
+    for your app.
+- Automatically scales your pods up and down, including scaling down to zero
+    active pods.
 
 ## Verification
 
 1. Run one of the followings commands to find the domain URL for your service.
    > Note: If your URL includes `example.com` then consult the setup instructions for
    > configuring DNS (e.g. with `sslip.io`), or [using a Custom Domain](https://knative.dev/docs/serving/using-a-custom-domain).
 
-    === "kubectl"
-
-        ```bash
-        kubectl get ksvc helloworld-python  --output=custom-columns=NAME:.metadata.name,URL:.status.url
-        ```
+### "kubectl"
```

Comment:
As above, these sections for kubectl and kn should be indented within the step on line 165. This also breaks numbering on line 197 (numbering restarts from 1). Consider restructuring.